lib/postIt/nodeFactory.js

```diff
@@ -36,6 +36,24 @@ const createTextNode = (width, height, label, index) => {
   return textNode
 }
 
+const createDeleteNode = (width, height, label, index, color) => {
+  const textNode = document.createElementNS(SVG_NS_URI, 'text')
+
+  textNode.setAttribute('alignment-baseline', 'middle')
+  textNode.setAttribute('text-anchor', 'middle')
+  textNode.setAttribute('x', width)
+  textNode.setAttribute('y', height)
+  textNode.setAttribute('font-size', '18')
+  textNode.setAttribute('font-weight', 'bold')
+  textNode.setAttribute('cursor', 'default')
+  textNode.setAttribute('data-index', index)
+  textNode.setAttribute('data-remove-by-index', index)
```

Comment:
Qui cambierei la gestione degli attributi. Dato che abbiamo già il `data-index` per indicare a quale post-it si far riferimento non aggiungerei due volte lo stesso valore. Ma dato che questo attributo ci fa da selettore possiamo semplicemente aggiungerlo ma vuoto.